kivy/tests/test_window_base.py

```diff
@@ -0,0 +1,16 @@
+from itertools import product
+
+from kivy.tests import GraphicUnitTest
+
+
+class WindowBaseTest(GraphicUnitTest):
+
+    def test_to_normalized_pos(self):
+        for x, y in product([0, 319, 50, 51], [0, 239, 50, 51]):
+            self.Window.system_size = (320, 240)
```

Comment:
Probably should save/restore original size after test with try/finally. So other tests use correct size.

Reply:
Agree. I'll add that change.